hummingbot/connector/exchange/bitfinex/bitfinex_api_order_book_data_source.py

```diff
@@ -87,7 +71,7 @@ def __init__(self, trading_pairs: Optional[List[str]] = None):
     async def fetch_trading_pairs() -> List[str]:
         try:
             async with aiohttp.ClientSession() as client:
-                async with client.get("https://api-pub.bitfinex.com/v2/conf/pub:list:pair:exchange", timeout=10) as response:
+                async with client.get(f"{BITFINEX_REST_URL}/conf/pub:list:pair:exchange", timeout=10) as response:
```

Comment:
The only change. The rest of the changes were introduced by the `isort`Git commit hook.